diff --git a/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index f5142af..4e9b9ed 100644
--- a/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/api-kit/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -202,7 +202,9 @@ exports.networks = [
     { chainId: 1666600000n, shortName: 'hmy-s0' },
     { chainId: 1666700000n, shortName: 'hmy-b-s0' },
     { chainId: 11297108099n, shortName: 'tpalm' },
-    { chainId: 11297108109n, shortName: 'palm' }
+    { chainId: 11297108109n, shortName: 'palm' },
+    { chainId: 42793n, shortName: 'etherlink' },
+    { chainId: 128123n, shortName: 'etherlink-testnet' },
 ];
 if (process.env.TEST_NETWORK === 'hardhat') {
     exports.networks.push({ shortName: 'local', chainId: 31337n });
