diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index a3c670d..6314df5 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -224,6 +224,8 @@ exports.networks = [
     { chainId: 1666700000n, shortName: 'hmy-b-s0' },
     { chainId: 11297108099n, shortName: 'tpalm' },
     { chainId: 11297108109n, shortName: 'palm' },
+    { chainId: 42793n, shortName: 'etherlink' },
+    { chainId: 128123n, shortName: 'etherlink-testnet' },
     { chainId: 88153591557n, shortName: 'arb-blueberry' }
 ];
 if (process.env.TEST_NETWORK === 'hardhat') {
